requirements/prod.in

```diff
@@ -9,19 +8,18 @@ django-cache-url==3.4.5
 django-cors-headers==4.3.1
 django-extensions==3.2.3
 django-mozilla-product-details==1.0.3
-django-picklefield==3.1
```

Comment:
Long unneeded?

Reply:
yeah, didn't see any use of it anywhere.